heareval/score.py

```diff
@@ -53,6 +54,28 @@ def label_to_binary_vector(label: List, num_labels: int) -> torch.Tensor:
     return binary_labels
 
 
+def validate_score_return_type(
+    ret: Union[float, int, Tuple[Tuple[str, Union[int, float]]]]
```

Comment:
When would we ever return an int? The return type for a score should always be float